docs/running-clusterfuzzlite/github_actions.md

```diff
@@ -64,24 +65,32 @@ on:
   pull_request:
     paths:
       - '**'
+permissions: read-all
 jobs:
   PR:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sanitizer: [address, undefined, memory]
```

Comment:
I think we should probably have a comment here explaining that this should only contain the fuzzers users want.

Reply:
done.